doc/docs/learn/create-an-app-with-monomer.md

```diff
@@ -206,10 +185,16 @@ func initRootCmd(
 ```
 
 ## Building the Application
-Before we can build our application, we need to add the following replace directive in `go.mod` and then run `go mod
+Before we can build our application, we need to add the following replace directives in `go.mod` and then run `go mod
 tidy`: 
 ```go
 github.com/ethereum/go-ethereum => github.com/joshklop/op-geth v0.0.0-20240515205036-e3b990384a74
+github.com/libp2p/go-libp2p => github.com/joshklop/go-libp2p v0.0.0-20240814165419-c6b91fa9f263
```

Comment:
I had to run `go get github.com/libp2p/go-libp2p` before this replace directive could be added. I'm assuming that's because the libp2p changes weren't directly added to monomer yet?

Reply:
Hmm thanks for raising this. Out of curiosity Did you run `go mod tidy` after adding the replace (and before running `go get`)? `go mod tidy` does `go get` for modules it can't find.

Reply:
Ran through the flow again and seems like I just missed a `go mod tidy`